usage/camera/installing.md

```diff
@@ -2,9 +2,9 @@
 
 ## Connecting the cameras
 
-The flex cable inserts into the connector situated between the Ethernet and HDMI ports, with the silver connectors facing the HDMI port. The flex cable connector should be opened by pulling the tabs on the top of the connector upwards then towards the Ethernet port. The flex cable should be inserted firmly into the connector, with care taken not to bend the flex at too acute an angle. The top part of the connector should then be pushed towards the HDMI connector and down, while the flex cable is held in place.
+The flex cable inserts into the connector labelled CAMERA on the Raspberry Pi, which is located between the Ethernet and HDMI ports. The cable must be inserted with the silver contacts facing the HDMI port. To open the connector, pull the tabs on the top of the connector upwards, then towards the Ethernet port. The flex cable should be inserted firmly into the connector, with care taken not to bend the flex at too acute an angle. To close the connector, push the top part of the connector towards the HDMI port and down, while holding the flex cable in place.
```

Comment:
Do you think it's worth adding some words describing how to do the same for the Pi Zero @andrum99 ?

Reply:
> Do you think it's worth adding some words describing how to do the same for the Pi Zero @andrum99 ?

Yes, but I someone else with knowledge of the board would need to do that. While I can guess based on images I can find online, the connector is different so the way it opens and closes might be different as well. I will leave that to someone else - either as part of this PR or a separate future PR.

Reply:
Just created #1709 so that it doesn't get forgotten about.